service/notification/NotificationProcessor.php

```diff
@@ -204,8 +224,7 @@ public function processNotification($unprocessedNotification)
                             // Moves order to canceled
                             $this->orderService->updateOrderState(
                                 $order,
-                                \Configuration::get('PS_OS_CANCELED'),
-                                $extraVars
```

Comment:
why removing the extravars here? In case the auth false notification contains a pspreference we can still update the order payment with it, can't we?

Reply:
Originally I removed the `extraVars` from the `updateOrderState` function so that the pspReference is updated in another function to have better single responsibility. That being said, I should have called this function in this specific case, which was added in this [commit](https://github.com/Adyen/adyen-prestashop/pull/177/commits/6bf71230405a7ffd227b4e00d386976e201e0526)